spec/finders/conversation_finder_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+describe ConversationFinder do
+  let!(:account) { create(:account) }
+  let!(:user_1) { create(:user, account: account) }
+  let!(:user_2) { create(:user, account: account) }
+  let!(:inbox) { create(:inbox, account: account) }
+  # rubocop:disable RSpec/LetSetup
+  let!(:inbox_member_1) { create(:inbox_member, user: user_1, inbox: inbox) }
+  let!(:inbox_member_2) { create(:inbox_member, user: user_2, inbox: inbox) }
+  let!(:conversation_1) { create(:complete_conversation, account: account, inbox: inbox, assignee: user_1) }
+  let!(:conversation_2) { create(:complete_conversation, account: account, inbox: inbox, assignee: user_1) }
+  let!(:conversation_3) { create(:complete_conversation, account: account, inbox: inbox, assignee: user_1, status: 'resolved') }
+  let!(:conversation_4) { create(:complete_conversation, account: account, inbox: inbox, assignee: user_2) }
+  # rubocop:enable RSpec/LetSetup
+
+  describe 'Filtering' do
```

Comment:
@subintp It would be good if `describe` is used point to a method the test is written for rather than the function its performing (which is already written in context) as it would be easier for an individual to understand just looking at the describe of the test which function the test is written for.

Reply:
Done.